Backend/app/routes/brand_dashboard.py

```
@@ -0,0 +1,1132 @@
+                
+                matches.append(CreatorMatchResponse(**match))
+        
+        return matches
+        
+    except Exception as e:
+        logger.error(f"Error fetching creator matches: {e}")
+        raise HTTPException(status_code=500, detail="Internal server error")
+
+@router.get("/creators/search")
+async def search_creators(
+    brand_id: str = Query(..., min_length=36, max_length=36, regex=r"^[a-fA-F0-9\-]{36}$", description="Brand user ID (UUID)"),
```

Ruff (0.13.1)

Unused function argument: `brand_id`

(ARG001)